setup.py

```diff
@@ -45,7 +45,7 @@ def main():
     install_requires=[
         'dill>=0.3.2',
     ],
-    python_requires='>=3.6, <3.9',
+    python_requires='>=3.6, <3.10',
```

Comment:
Seems to work fine on `3.10.6` as well, so it's probably better to just set `<3.11` (as that's the current preview version) for now.

Reply:
+1. it looks good to pin 3.11 for now.